.gitignore

```
@@ -1,12 +1,9 @@
-.DS_Store
-node_modules
-TODO.md
-lib
-docs/.vuepress/dist
-examples/**/build.js
-types/typings
-types/test/*.js
-explorations
+/docs/.vuepress/dist
+/examples/**/build.js
+/node_modules
```

```suggestion
node_modules
```

As `node_modules` should always be ignored.

---

Fixed! I think this is same for Vuex 4 so I'll make sure to update that one as well when we merge this.

---

Oh thanks. I had overlooked it 😣